integration-cli/docker_cli_commit_test.go

```diff
@@ -116,9 +116,9 @@ func (s *DockerSuite) TestCommitChange(c *check.C) {
 		"--change", "ENV test 1",
 		"--change", "ENV PATH /foo",
 		"--change", "LABEL foo bar",
-		"--change", "CMD [\"/bin/sh\"]",
+		"--change", `"CMD [""/bin/sh""]"`,
```

Comment:
Oh, sorry I got that backwards. It used to be a ListOpts, so I think that
is the difference. Does it work if you go back to using the list opts, or
is it broken with both types ?
On Jun 8, 2016 11:38 AM, "Yong Tang" <notifications@github.com> wrote:

> In integration-cli/docker_cli_commit_test.go
> <https://github.com/docker/docker/pull/23309#discussion_r66279789>:
>
> > @@ -116,9 +116,9 @@ func (s *DockerSuite) TestCommitChange(c *check.C) {
> >  		"--change", "ENV test 1",
> >  		"--change", "ENV PATH /foo",
> >  		"--change", "LABEL foo bar",
> > -		"--change", "CMD [\"/bin/sh\"]",
> > +		"--change", `"CMD [""/bin/sh""]"`,
>
> @dnephin <https://github.com/dnephin> At the moment --change use
> StringSliceVarP. I searched the web and most of the same error message bare
> " in non-quoted-field" points to golang's csv package. I don't know
> enough detail about the implementation in cobra though.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/docker/docker/pull/23309/files/89edc4e36c371d04c5d65ff1d330f13d5d4fdc52#r66279789>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAa_REm_dIKPUKsYjV_aM_a-4Cnk7IvEks5qJuHogaJpZM4Iu_PI>
> .
>